torch/distributed/distributed_c10d.py

```diff
@@ -1465,6 +1466,186 @@ def reduce_scatter(output,
         work.wait()
 
 
+def all_to_all_single(output,
+                      input,
+                      output_split_sizes=None,
+                      input_split_sizes=None,
+                      group=group.WORLD,
+                      async_op=False):
+    """
+    Each process splits input tensor and then scatters the split list
+    to all processes in a group. Then concatenate the received tensors from all
+    the processes in the group and return single output tensor.
+
+    Arguments:
+        output (Tensor): Gathered cancatenated output tensor 
+        input (Tensor): Input tensor to scatter 
+        output_split_sizes: (list[Int], optional): Output split sizes for dim 0
+            if specified None or empty, dim 0 of `output` tensor must divide
```

Comment:
To make `output` (and other variables and functions) show as code instead of italic, we need to use double quote: \`\`output\`\`